### DISCUSSION TITLE
空きのSRAMを教えて下さい

### DISCUSSION THREAD

**Discussion by Isaac45132** [ngmansion/FE8N#68] [Q&A] [answered]
自軍
AIカウンタ１=戦技
AIカウンタ２＝疾風迅雷等のスキル
AI1=-
AI2=-
AI3=使ってますか？
AI4=使ってますか？

戦技に使われているAI1カウンターを使いたいのですが、戦技を一切使わない場合は
利用できますか？

**Comment by aera-ngmansion** [comment, marked as answer]
今の戦技はその領域を使ってません
具体的には、敵が戦技を使えるようになる更新で、使う領域を変えたと思います
